src/main/kotlin/io/github/kryszak/gwatlin/api/gamemechanics/model/facts/FactWithNoTypeSerializer.kt

```diff
@@ -0,0 +1,44 @@
+package io.github.kryszak.gwatlin.api.gamemechanics.model.facts
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.json.JsonDecoder
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.modules.SerializersModule
+
+/**
+ * Register into [SerializersModule] to support Facts in JSON
+ * without type property. It's likely an error in API implementation,
+ * see e.g. [Nefarious Favor](https://api.guildwars2.com/v2/skills?ids=40813&v=latest).
+ *
+ */
+class FactWithNoTypeSerializer : KSerializer<Fact> {
```

Comment:
This class should be placed in `http/serializers` package, not api package

Reply:
Right, moved 👍